Anatoly Shlom/css/style.css

```diff
@@ -0,0 +1,113 @@
+
+/* Приводим браузерные стили в порядок */
+@import url('normalize.css');
+
+/* Добавляем необходимый шрифт*/
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700');
+* {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+    color: #ffffff;
+    padding: 0;
+    margin: 0;
+}
+
+/*Добавляем фон и немножко кросбраузерности)*/
+html { 
+    background: url("../img/bg.png") no-repeat center center fixed; 
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+
+/*оформляем сам виджет*/
+main {
+    background: no-repeat;
+    background-image: url("../img/main_bg.png");
+    background-size: cover; 
+    margin: 7% 11% 7% 11%;
+    padding: 10% 0 0 0;
+}
+
+/*верхняя часть виджета*/
+.wdata {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-start;
+    margin:0 10% 0 0;
+}
+
+/*секция с текущей датой*/
+.w_date {
+    margin: 20px 20px 0 0 ;
+}
+.cur_week_day {
+    font-size: 30px;
+}
+.cur_date {
+    font-size: 14px;
+}
+
+/*секция с температурой,локацией,ветром*/
+.info {
+    display: table-cell;
```

Comment:
why did you add this property to all flex-children?

Reply:
Fixed